pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -147,3 +147,9 @@ svg {
         }
     }
 }
+
+// Description sometimes shows up when empty, but this adds space to a grid,
+// causing extra space at the bottom
+.pf-c-alert__description:empty {
+    display: none;
+}
```

Comment:
After changing the fonts, I noticed the log in alert on the overview page was misaligned, as it had an empty description. I couldn't see a description being specified, so I was wondering if this is a PF4 issue or not. Unsure.

Ideally, if we could fix this in the JSX (`pkg/systemd/overview.jsx`, around line 140), that would be great. However, if this is a bug, we need to get it fixed upstream (in the JS). Meanwhile, this CSS will remove it from the layout if it's empty (so there won't be a gap, padding, etc.).

Reply:
Can you specify the conditions where there was not content in the alert apart from the titlle? I took a look to our code and I was not able to identify such case :( 
Btw it's not an upstream issue, not providing the Alert body will not render the pf-c-alert__descriotion.

Reply:
It looks like this:

![image](https://user-images.githubusercontent.com/10246/126333745-1e3de52f-aaf8-4545-94c4-99adf35a7df8.png)

See how the title is misaligned due to having an empty body for the alert?

Reply:
Here's what the DOM looks like:

![Screenshot from 2021-07-20 15-38-56](https://user-images.githubusercontent.com/10246/126333967-6d3eeb65-9696-476b-a24d-9b9f28bebab3.png)

I have the offending description body selected to highlight it.

Reply:
This needs to be fixed in JS, we currenty past the last_login_message as an empty string to the Alert component, therefore you see this.

Reply:
@KKoukiou: Do you think the empty `last_login_message` needs to be fixed within this PR?

Reply:
As I mention in the comment above this is the way to fix this. 
Firstly: PF4 overrides always should have a upstream issue which is missing here
Secondly: This looks like that needs to be fixed in JS instead.

Reply:
We talked in IRC; this should be moved to the local CSS, as it probably is only triggered by the JS locally. Ideally, the JavaScript would be fixed, but it's a bit out of scope of this PR and I can't do that... I'd need someone else to more properly fix it.

Reply:
@garrett this can be properly fixed with:
``` diff
diff --git pkg/systemd/overview.jsx pkg/systemd/overview.jsx
index 1aef9178d..53bdff620 100644
--- pkg/systemd/overview.jsx
+++ pkg/systemd/overview.jsx
@@ -143,6 +143,11 @@ class LoginMessages extends React.Component {
 
         const last_log_item = <span id='last-login'><strong>{_("Last login:")}</strong> {last_login_message}</span>;
         const last_fail_item = <span id='last-failed-login'><strong>{_("Last failed login:")}</strong> {last_fail_message}</span>;
+        const child = [];
+        if (last_login_message && fail_count_message);
+            child.append(last_log_item);
+        if (last_fail_message)
+            child.append(last_fail_item);
 
         return (
             <Alert id='login-messages'
@@ -152,8 +157,7 @@ class LoginMessages extends React.Component {
                    actionClose={<AlertActionCloseButton onClose={this.close} />}
                    title={fail_count_message || last_log_item}
             >
-                {last_login_message && fail_count_message && last_log_item}
-                {last_fail_message && last_fail_item}
+                {child.length ? child : null}
             </Alert>
         );
     }
```

but it will conflict with Jelles PR so let's just remove and let him fix it.